Kickstarter-iOS/ViewModels/RootViewModel.swift

```diff
@@ -157,6 +160,7 @@ internal final class RootViewModel: RootViewModelType, RootViewModelInputs, Root
     self.selectedIndex =
       Signal.combineLatest(
         .merge(
+          self.viewDidLoadProperty.signal.mapConst(0),
```

Comment:
This is so that by default it is tapped at least once (thanks to `viewDidLoad`). This should be a proper behavior.